medical_lab/models/medical_lab_test_criterion.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2012-2013 Federico Manuel Echeverri Choux
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class MedicalLabTestCriterion(models.Model):
+
```

Comment:
Interesting. I always wondered that, as PEP8 like you said states to leave a blank line, however I've seen all (most) underscore prefixed attrs grouped with no space between them and the class declaration in Odoo. So if there's just 1 attr, leave a blank line, and if there's a group of underscored attrs, then no blank line? Or should I just always put a blank line in between?